### ISSUE TITLE
2.0.2 doesn't work anymore

### ISSUE THREAD

**Issue by themilkman** [TheSecEng/ColoredComments#5]
Hi @TheSecEng 

I have trouble setting up ne latest version (I hope it is not a duplicate of #4):

I use this minimal configuration (esp. to eliminate the risk of JSON errors)

```
{
  "prompt_new_color_scheme": true,
  "continued_matching": false,
  "tags": {
    "TODO": {
      "identifier": "TODO",
      "color": {
        "name": "todo",
        "foreground": "#ff9900",
        "background": "rgba(1, 22, 38, 0.1)"
      }
    }
  }
}
```

which results in errors like this:

```
raceback (most recent call last):
  File "/opt/sublime_text/Lib/python33/sublime_plugin.py", line 1314, in run_
    return self.run(edit)
  File "/home/mysweethome/.config/sublime-text-3/Installed Packages/Colored Comments.sublime-package/colored_comments.py", line 32, in run
  File "/home/mysweethome/.config/sublime-text-3/Installed Packages/Colored Comments.sublime-package/color_manager.py", line 110, in create_user_custom_theme
  File "./python3.3/json/__init__.py", line 316, in loads
  File "./python3.3/json/decoder.py", line 351, in decode
  File "./python3.3/json/decoder.py", line 369, in raw_decode
ValueError: No JSON object could be decoded
Traceback (most recent call last):
  File "./python3.3/json/decoder.py", line 367, in raw_decode
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/sublime_text/Lib/python33/sublime_plugin.py", line 1314, in run_
    return self.run(edit)
  File "/home/mysweethome/.config/sublime-text-3/Installed Packages/Colored Comments.sublime-package/colored_comments.py", line 32, in run
  File "/home/mysweethome/.config/sublime-text-3/Installed Packages/Colored Comments.sublime-package/color_manager.py", line 110, in create_user_custom_theme
  File "./python3.3/json/__init__.py", line 316, in loads
  File "./python3.3/json/decoder.py", line 351, in decode
  File "./python3.3/json/decoder.py", line 369, in raw_decode
ValueError: No JSON object could be decoded

```

I tried several UI Color schemas and tried to Regenerate them using the command.
Usually I use the toxin color schema.

Any hints what's I could do?

Thanks in advance and again thanks for your work on this plugin!

**Comment by TerminalFi** [comment]
> toxin

Thanks for the issue, will look into this today. I‘ve ran into a few issues with the regenerate command that I wish I caught before release. I’ll be fixing these today and it should resolve this issue as welll as #4

**Comment by TerminalFi** [comment]
@themilkman Does the latest version resolve this?

**Comment by themilkman** [comment]
Thanks, I'll try it out tomorrow and give you feedback asap!

**Comment by themilkman** [comment]
Unfortunately it does not work yet (with various Color Schemes). I also tried to disable A File Icon to isolate the error below - with no success.
Also I noticed a 100% CPU usage and the custom-settings view (right side) seems to reload constantly.


```
Traceback (most recent call last):
  File "/opt/sublime_text/Lib/python33/sublime_plugin.py", line 1314, in run_
    return self.run(edit)
  File "/home/my_sweet_home/.config/sublime-text-3/Installed Packages/Colored Comments.sublime-package/colored_comments.py", line 115, in run
  File "/home/my_sweet_home/.config/sublime-text-3/Installed Packages/Colored Comments.sublime-package/color_manager.py", line 93, in create_user_custom_theme
  File "./python3.3/json/__init__.py", line 316, in loads
  File "./python3.3/json/decoder.py", line 351, in decode
  File "./python3.3/json/decoder.py", line 369, in raw_decode
ValueError: No JSON object could be decoded
```

**Comment by TerminalFi** [comment]
> Unfortunately it does not work yet (with various Color Schemes). I also tried to disable A File Icon to isolate the error below - with no success.
> Also I noticed a 100% CPU usage and the custom-settings view (right side) seems to reload constantly.
> 
> ```
> Traceback (most recent call last):
>   File "/opt/sublime_text/Lib/python33/sublime_plugin.py", line 1314, in run_
>     return self.run(edit)
>   File "/home/my_sweet_home/.config/sublime-text-3/Installed Packages/Colored Comments.sublime-package/colored_comments.py", line 115, in run
>   File "/home/my_sweet_home/.config/sublime-text-3/Installed Packages/Colored Comments.sublime-package/color_manager.py", line 93, in create_user_custom_theme
>   File "./python3.3/json/__init__.py", line 316, in loads
>   File "./python3.3/json/decoder.py", line 351, in decode
>   File "./python3.3/json/decoder.py", line 369, in raw_decode
> ValueError: No JSON object could be decoded
> ```

Thanks! It looks like it’s having issues reading your current color scheme. I’ll do some debugging on that specific one.

**Comment by TerminalFi** [comment]
Fixed in [v2.0.4](https://github.com/TheSecEng/ColoredComments/tree/2.0.4)
